dev-libs/libdbi/libdbi-0.9.0.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="Database-independent abstraction layer written in C"
+DESCRIPTION="libdbi implements a database-independent abstraction layer in C, similar to the DBI/DBD layer in Perl. Writing one generic set of code, programmers can leverage the power of multiple DBs and multiple simultaneous DB connections by using this framework."
```

Comment:
Keep DESCRIPTION lengt to mas 100 characters:

https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#general-tips